src/sentry/static/sentry/app/views/ruleEditor/index.jsx

```diff
@@ -162,32 +176,30 @@ const RuleEditor = createReactClass({
             <h6>{t('Rule name')}:</h6>
             <div className="control-group">
               <input
-                ref="name"
                 type="text"
                 className="form-control"
                 defaultValue={name}
                 required={true}
                 placeholder={t('My Rule Name')}
+                onChange={e => this.updateRule('name', e.target.value)}
```

Comment:
Let's start following this convention: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-handler-names.md